### PR TITLE
4 New Elements!

### PR DESCRIPTION
**Pull request by ajawsome4ev**
Plant Grows at a 50% chance charcoal burns very slowly generates lots of heat (great for grilling) and ash is burnt charcoal and is naturally warm

### CODE REVIEW

**Comment by l2wilson94** [comment]
The elements look great!

I think we should move the elements from the `CustomElements.js` file into one of the other files. Maybe `Sandbox.js`? We already have Glass element, so we'd just need to move the charcoals and ash. If you do that, I'd be happy to accept the pull request.

Thank you!

**Comment by ajawsome4ev** [comment]
okay cool

**Comment by ajawsome4ev** [comment]
the thing is that sandbox is getting very long and hard to read maybe a scroll bar is in order?
?

**Comment by ajawsome4ev** [comment]
the thing is that sandbox is getting very long and hard to read maybe a scroll bar is in order?
?

**Comment by l2wilson94** [comment]
Great, thanks! And that's a good point! I've put an issue for fixing scroll, so I will try to fix that eventually. For the moment, I wonder if we should separate them into smaller categories. What do you think? I think Plant should go into the Life category probably.

Also, we still need to remove the elements from CustomElements.js so that it is nice and empty for people to try out the language if they want. Would you like to do it? Otherwise, I'd be happy to clean it up and accept the changes.

**Comment by ajawsome4ev** [comment]
yeah i  will move plant into the life category and fix it i also cleared the custom elements

**Comment by ajawsome4ev** [comment]
its done now

**Comment by ajawsome4ev** [comment]
Fixed it finnally

File: Source/Elements/Sandbox.js [hunk 1]
```diff
@@ -408,6 +408,53 @@ element Glass {
...
+		[HOT]: [FlamingCharcoal, 0.12],
+	})
+	
+	mimic(Temperature)
+	mimic(Solid)
+}
+
+element FlamingCharcoal {
+	colour "#2e362d"
+	emissive "2e362d"
```
Comment (l2wilson94) [inline Source/Elements/Sandbox.js:427]: This line is causing an error, it needs a '#' before the colour. Then I think it's good to go!

**Review by l2wilson94 — CHANGES_REQUESTED** [review]
Almost there! :)

**Review by l2wilson94 — APPROVED** [review]

**Comment by l2wilson94** [comment]
Hooray!!

**Pull request merged**